physionet-django/user/templates/user/settings.html

```diff
@@ -13,12 +13,11 @@
   <div class="row row-offcanvas row-offcanvas-right">
     <div class="col-6 col-md-3 sidebar-offcanvas sett-side" id="sidebar">
       <div class="list-group">
-        {% with 'Profile Password Emails Username Cloud ORCID Credentialing Agreements' as setting_options %}
-            {% for setting in setting_options.split %}
-              {% url 'edit_'|add:setting|lower as settings_url %}
-              <a href={{ settings_url }} class="list-group-item list-group-item-action{% if settings_url == request.path %} active{% endif %}">{{ setting }}</a>
-            {% endfor %}
-        {% endwith %}
+      {% setting_options sso_enabled as setting_opt %}
```

Comment:
You are now using a `simple_tag` which semantically is not the way to go (I think). Now that I look at it we probably should use an [inclusion tag](https://docs.djangoproject.com/en/4.0/howto/custom-template-tags/#inclusion-tags) (tag that renders a template).

It would work like this:
1. We register an `inclusion_tag` called `settings_tabs` that creates a context containing the settings options (array of strings, with/without 'Password' depending on the envvar)
2. The tag renders a template called e.g. `settings_tabs.html`
3. The template contains the loop that renders stuff:
```
{% for setting in setting_opt %}
    {% url 'edit_'|add:setting|lower as settings_url %}
    <a href={{ settings_url }} class="list-group-item list-group-item-action{% if settings_url == request.path %} active{% endif %}">{{ setting }}</a>
{% endfor %}
 ```

Reply:
It's cleaner imho.